config/application.rb

```diff
@@ -83,6 +83,9 @@ class Application < Rails::Application
       config.url_host = ENV['URL_HOST'] || ''
     end
 
+    # Specify the log level
+    config.log_level = ENV["RAILS_LOG_LEVEL"].present? ? ENV['RAILS_LOG_LEVEL'].to_sym : :debug
```

Comment:
This change presents some odd changes. This changes the default log level to debug for all existing deployments.

I think you're better off moving this line to the `production.rb` file and changing the else to be `:info`

Reply:
Good hint!